browser/brave_content_browser_client.cc

```diff
@@ -563,6 +570,20 @@ bool BraveContentBrowserClient::CanCreateWindow(
       container_type, target_url, referrer, frame_name, disposition, features,
       user_gesture, opener_suppressed, no_javascript_access);
 
+#if BUILDFLAG(ENABLE_REQUEST_OTR)
+  // If the user has requested going off-the-record in this tab, don't allow
```

Comment:
is there a test for this? I didn't see one

Reply:
added tests